spectator/percentile_distribution_summary.cc

```diff
@@ -0,0 +1,38 @@
+#include "percentile_distribution_summary.h"
+#include "percentile_bucket_tags.inc"
+
+using nanos = std::chrono::nanoseconds;
+
+namespace spectator {
+
+PercentileDistributionSummary::PercentileDistributionSummary(
+    spectator::Registry* registry, spectator::IdPtr id) noexcept
```

Comment:
Looks like the min/max restrictions are present on PercentileTimer, but not here.

Reply:
I wasn't sure whether it was needed here. Added in #20 

Thanks.

Reply:
It isn't used as much with distribution summaries, but I've been trying to have them as consistent as possible.